### ISSUE TITLE
Motor runs every time in same direction

### ISSUE THREAD

**Issue by mmmmmmmmmmmmmmm2** [MaslowCNC/Mechanics#100]
Hi All please help me out, one of the motors is always running in the same direction, if i push up it goes up, if i push down it goes again up. and if i let it go to center it is every time somewhere else, and doesn't stop anywhere. what do i need to do...

**Comment by JDogHerman** [comment]
Most likely the H-Bridge is dead on the controller card. 

You could eliminate other remote possibilities that cabling  by swapping the Left and Right motor wires running to the card.

**Comment by BarbourSmith** [comment]
I think this sounds like an encoder issue to me. If the motor can't read the encoder value it will always spin in one direction. What do you see happen when you press Actions -> Test motors / encoders?

**Comment by mmmmmmmmmmmmmmm2** [comment]
Hi All,
tested the motors, it says testing L motor direction 1 Fail , direction 2 Fail, and that for L / R and Z
See attached photo, it is brand new, must be something i´m doing wrong

![20200205_191115](https://user-images.githubusercontent.com/52886167/73870634-b9320980-484c-11ea-8c35-6409785faca8.jpg)

**Comment by BarbourSmith** [comment]
Did you close the issue because you found the solution?

**Comment by mmmmmmmmmmmmmmm2** [comment]
No i didn't, still not solve it ..

Pushed wrong button

Op wo 5 feb. 2020 23:14 schreef BarbourSmith <notifications@github.com>:

**Comment by BarbourSmith** [comment]
Ok, let's figure out what is going on.

Do you see the motors move when running the test?

**Comment by mmmmmmmmmmmmmmm2** [comment]
Hi all,

No, they do not move, when testing


But when i push manually it moves..

Op do 6 feb. 2020 23:34 schreef BarbourSmith <notifications@github.com>:

**Comment by BarbourSmith** [comment]
Gotcha. When all three motors fail the test the issue is almost always with the power supply. Does the power supply have a blue light on it? Does the green light next to the plug come on when you plug the power supply into the controller board?

**Comment by mmmmmmmmmmmmmmm2** [comment]
Good,

Correct, only two green lights.. no blue lights

Op vr 7 feb. 2020 07:14 schreef BarbourSmith <notifications@github.com>:

**Comment by BarbourSmith** [comment]
Where are the the green lights? Are they both on the controller board? Can you post a picture of the controller board and the power supply?

**Comment by mmmmmmmmmmmmmmm2** [comment]
See attached photos

One of controllerboard and of the adapter


Op vr 7 feb. 2020 17:36 schreef BarbourSmith <notifications@github.com>:

**Comment by BarbourSmith** [comment]
I don't think the photos actually attached 😢

**Comment by mmmmmmmmmmmmmmm2** [comment]
Mmmm, strange..

Will send them in 2 emails

Op vr 7 feb. 2020 23:01 schreef BarbourSmith <notifications@github.com>:

**Comment by mmmmmmmmmmmmmmm2** [comment]
Number 2

Op vr 7 feb. 2020 23:01 schreef BarbourSmith <notifications@github.com>:

**Comment by davidelang** [comment]
they did not come through, try adding them through the web interface.

David Lang

**Comment by mmmmmmmmmmmmmmm2** [comment]
Okay, here they are
![Uploading 20200207_175711.jpg…]()

**Comment by BarbourSmith** [comment]
I'm still not seeing the picture :( 

![image](https://user-images.githubusercontent.com/9359447/74106459-05ff3400-4b1c-11ea-8b90-5de6f74759f6.png)

Let's move this conversation into the forums...I think we'll have less technical trouble there, you will get faster responses from more people, and that is really the right place for the conversation since this is about solving a particular issue instead of proposing a change to the design of the machine

**Comment by BarbourSmith** [comment]
The forums are https://forums.maslowcnc.com/. You can login with your github login so no need to make an account or anything

**Comment by mmmmmmmmmmmmmmm2** [comment]
![20200207_175711](https://user-images.githubusercontent.com/52886167/74106590-877cb380-4b68-11ea-9417-5a84a9d448a2.jpg)
First

**Comment by mmmmmmmmmmmmmmm2** [comment]
![20200207_175700](https://user-images.githubusercontent.com/52886167/74106600-9cf1dd80-4b68-11ea-8737-1418ef4d4360.jpg)
Second photo

**Comment by BarbourSmith** [comment]
Nice! Ok everything looks good there. And we are still seeing the motor move when you manually command them to move, but no movement at all during the test?

**Comment by mmmmmmmmmmmmmmm2** [comment]
Yes, no movement during test, but can do manual but always same direction

**Comment by BarbourSmith** [comment]
This is totally like anything I have seen before. The test should be overriding the controls in a way which makes the motor move if it has power...but the pictures and the fact that you can manually move it mean we have power

**Comment by BarbourSmith** [comment]
Is the behavior the same on all the motors? The only thing I can think could be a bad connection in one of the connectors. Maybe let's try unplugging them all and re-plugging each motor very firmly

**Comment by mmmmmmmmmmmmmmm2** [comment]
Strange isn't? Is an idea to set up software again?

**Comment by BarbourSmith** [comment]
So strange 😲 

It could be worth setting up the software again, but this feels like a hardware issue to me

**Comment by BarbourSmith** [comment]
I think checking the connections on the cables is the best place to start. I would unplug each motor and re-plug them firmly

**Comment by mmmmmmmmmmmmmmm2** [comment]
It is new bought it end December , coming 2 weeks i'm not near my cnc, if you have any suggestions please let me know

**Comment by BarbourSmith** [comment]
Did you try unplugging and re-plugging the cables to the motors? 

Have you contacted MakerMade? Do they have any suggestions?

**Comment by mmmmmmmmmmmmmmm2** [comment]
Haven't tried to switch the motors but tried several times the connection. Have no idea who to contact assume there is some kind of warranty..

**Comment by BarbourSmith** [comment]
They should send you a free replacement for anything which is not working...the tricky thing is I'm not even sure what to ask them for because I'm not sure what is going on. Maybe ask for a new arduino and controller board to be safe?

You can find them in the forums by posting your issue and tagging them as @makermadecnc they usually respond there pretty quickly
